extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -165,5 +168,27 @@ export class MethodModelingViewProvider extends AbstractWebviewViewProvider<
         }
       }),
     );
+
+    this.push(
+      this.modelingStore.onDbOpened(async () => {
+        if (this.webviewView) {
```

Comment:
It seems like this is already handled by `postMessage` using optional chaining. Does it still make sense to have this check here for a simple call like this?